### ISSUE TITLE
Try increasing processing size of uploaded images

### ISSUE THREAD

**Issue by as2388** [oscarkey/extrusionfinder#49]
This may identify details better

**Comment by Veedrac** [comment]
I don't think this is worth doing, especially if it costs in processing time. The scaling down actually helps remove noise artifacts IMO.

**Comment by as2388** [comment]
If you have evidence to back up your claim then sure.
